### PR TITLE
Updated support for electron 26

### PR DESCRIPTION
**Pull request by mikeyyyzhao**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,18937 @@
+{
```
Comment (amaury1729) [inline package-lock.json:1]: please remove this file

File: package.json [hunk 2]
```diff
@@ -47,17 +47,17 @@
...
     "@amaurym/config": "^1.3.5",
     "@types/jest": "^25.2.3",
-    "electron": "^22.0.0",
+    "electron": "^27.0.0",
     "jest": "^26.0.1",
     "rimraf": "^3.0.2",
     "standard-version": "^8.0.0",
     "ts-jest": "^26.0.0",
-    "typedoc": "^0.17.7",
+    "typedoc": "^0.25.2",
```
Comment (amaury1729) [inline package.json:55]: Let's only update electron in this PR

File: package.json [hunk 3]
```diff
@@ -47,17 +47,17 @@
...
     "typedoc-plugin-markdown": "^2.2.17",
     "typedoc-plugin-no-inherit": "^1.1.10",
     "typescript": "^4.6.2"
   },
   "peerDependencies": {
-    "electron": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0 || ^22.0.0"
+    "electron": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0 || ^22.0.0 || ^26.0.0"
```
Comment (amaury1729) [inline package.json:61]:
```suggestion
    "electron": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0 || ^22.0.0 || ^26.0.0 || ^27.0.0"
```

File: package.json [hunk 4]
```diff
@@ -47,17 +47,17 @@
   "devDependencies": {
```
Comment (amaury1729) [inline package.json:47]: Can you also update to Electron 27 in examples/pacakge.json?

**Review by amaury1729 — COMMENTED** [review]
Thanks! Still some stuff to fix before merging

**Comment by amaury1093** [comment]
Superseded by https://github.com/max-mapper/menubar/pull/476

**Pull request closed without merging**